### PR TITLE
Status command

### PR DESCRIPTION
**Pull request by ananya21**

### CODE REVIEW

**Review by Lin-Shuang-Shuang — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/SetStatusCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddCommand
+     * and returns an AddCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public SetStatusCommand parse(String args) throws ParseException {
+        String status = args.split(" of")[0].trim();
+        String taskAndProject = args.split(" of")[1].trim();
+        String taskName = taskAndProject.split("in ")[0].trim();
+        String projectName = taskAndProject.split("in ")[1];
```
Comment (modembcc) [inline src/main/java/seedu/address/logic/parser/SetStatusCommandParser.java:27]: This doesn't match with the command word in https://github.com/AY2324S2-CS2103T-W08-4/tp/blob/master/docs/UserGuide.md.

File: src/main/java/seedu/address/logic/parser/SetStatusCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        if (projectName.length() == 0) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, SetStatusCommand.MESSAGE_USAGE));
+        }
+        Person project = new Person(ParserUtil.parseName(projectName));
+        Task newTask = new Task(taskName);
+        return new SetStatusCommand(status, newTask, project);
```
Comment (modembcc) [inline src/main/java/seedu/address/logic/parser/SetStatusCommandParser.java:33]: What if we create 2 types of constructor instead, 1 for set status of project and another ofr set status of task.

**Review by modembcc — CHANGES_REQUESTED** [review]
There are some parts I'm confused about.

**Review by modembcc — APPROVED** [review]
It looks nice. Good job!

**Pull request merged**